README.md

```diff
@@ -5,7 +5,7 @@
 
 > ⚠️ This project is experimental and not suitable for production use. ⚠️
```

Comment:
I wouldn’t call this experimental any more, let’s change this to say that the api is still unstable.